gtfs-realtime/spec/en/trip-updates.md

```diff
@@ -6,7 +6,7 @@ There should be **at most** one trip update for each scheduled trip. In case the
 
 ## Stop Time Updates
 
-A trip update consists of one or more updates to vehicle stop times, which are referred to as [StopTimeUpdates](reference.md#StopTimeUpdate). These can be supplied for past and future stop times. You are allowed, but not required, to drop past stop times. When doing this, be aware that you shouldn't drop a past update if it refers to a trip that isn't yet scheduled to have finished (i.e. it finished ahead of schedule) as otherwise it will be concluded that there is no update on this trip.
+A trip update consists of one or more updates to vehicle stop times, which are referred to as [StopTimeUpdates](reference.md#StopTimeUpdate). These can be supplied for past and future stop times. You are allowed, but not required, to drop past stop times. When doing this, be aware that you shouldn't drop a past StopTimeUpdate if it refers to stop that a vehicle isn't yet scheduled to have passed (i.e. running ahead of schedule) as otherwise it will be concluded that there is no update for this stop.
```

Comment:
"a vehicle isn't yet scheduled to have passed" -- this blows up my mind, can we somehow re-formulate it?